packages/components/src/range-control/style.scss

```diff
@@ -0,0 +1,8 @@
+// Keeping this className style rule as it is currently being
+// used OUTSIDE of the RangeControl component.
```

Comment:
Bad :)

Reply:
I did a quick search and I noticed that it's only used in FontSizePicker. 
Maybe we can just move the style there and rename the class to follow our guidelines `components-font-size-picker__number`

Reply:
Sounds good to me! I'll do that now